Test failures on macOS 11.6.4 (big sur) when installing perl-5.34.1-RC2

OS: macOS 11.6.4, Intel CPU.

Failures in only tests of "podlators". 

```
# perlbrew install -j2 perl-5.34.1-RC2                      

Test Summary Report
-------------------
../cpan/podlators/t/general/basic.t                                (Wstat: 256 Tests: 15 Failed: 1)
  Failed test:  11
  Non-zero exit status: 1
../cpan/podlators/t/text/color.t                                   (Wstat: 1280 Tests: 11 Failed: 5)
  Failed tests:  3, 5, 7, 9, 11
  Non-zero exit status: 5
Files=2676, Tests=1185872, 637 wallclock secs (134.46 usr 25.06 sys + 746.54 cusr 108.41 csys = 1014.47 CPU)
Result: FAIL
make: *** [test_harness] Error 6
```

```
# cc -v
Apple clang version 13.0.0 (clang-1300.0.29.30)
Target: x86_64-apple-darwin20.6.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin

# xcode-select -v
xcode-select version 2384.
```

--- a/lib/App/perlbrew.pm
+++ b/lib/App/perlbrew.pm
@@ -1675,7 +1675,7 @@ INSTALL
         $cmd = "($cmd) >> '$self->{log_file}' 2>&1 ";
     }
 
-    delete $ENV{$_} for qw(PERL5LIB PERL5OPT AWKPATH);
+    delete $ENV{$_} for qw(PERL5LIB PERL5OPT AWKPATH NO_COLOR);
 
     if ($self->do_system($cmd)) {
         my $newperl = $self->root->perls ($installation_name)->perl;

